spec/forms/steps/other_parties/address_details_form_spec.rb

```diff
@@ -71,5 +89,89 @@
         end
       end
     end
+    context 'when split address' do
+      let(:split_address_result) { true }
+
+      let(:address_data) do
+        {
+          address_line_1: address_line_1,
+          address_line_2: address_line_2,
+          town: town,
+          country: country,
+          postcode: postcode
+        }
+      end
+
+      context 'validations on field presence unless `unknown`' do
+        it { should validate_presence_of(:address_line_1) }
```

Comment:
Shouldn't this be:

```suggestion
        it { should validate_presence_unless_unknown_of(:address_line_1) }
```

Same for `postcode`.

Also, revisit the respondent spec in case the same has happened.